lib/webview-helpers.js

```diff
@@ -367,7 +363,7 @@ helpers.getWebViewsMapping = async function getWebViewsMapping (adb, {
   androidDeviceSocket = null,
   ensureWebviewsHavePages = true,
   webviewDevtoolsPort = null,
-  enableWebviewDetailsCollection = null,
+  enableWebviewDetailsCollection = true,
```

Comment:
Reminder: Update the readme for UIA2 and Espresso after this PR is merged and published

Reply:
Thanks. I've prepared PRs
https://github.com/appium/appium-uiautomator2-driver/pull/467
https://github.com/appium/appium-espresso-driver/pull/677